bliss/models/decoder.py

```diff
@@ -453,14 +446,14 @@ def forward(self, locs, fluxes, star_bools):
         sources = expanded_psf * rearrange(fluxes, "np ms nb -> np ms nb 1 1")
         sources *= rearrange(star_bools, "np ms 1 -> np ms 1 1 1")
 
-        return self.tiler(locs, sources)
+        return sources
 
 
-class GalaxyTileDecoder(nn.Module):
+class GalaxyDecoder(nn.Module):
     def __init__(self, tile_slen, ptile_slen, n_bands, galaxy_model: GalaxyModel):
         super().__init__()
         self.n_bands = n_bands
-        self.tiler = Tiler(tile_slen, ptile_slen)
+        self.tiler = TileRenderer(tile_slen, ptile_slen)
```

Comment:
since you are changing `GalaxyTileDecoder` to `GalaxyDecoder` it will also make sense to replace `ptile_slen` with `slen`, then there is no reference to tiles for any of these three clases. I think this will make the classes conceptually cleaner.